src/test/java/org/folio/circulation/support/JsonPropertyWriterTest.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.circulation.support;
+
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.folio.circulation.support.JsonPropertyWriter.writeByPath;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.allOf;
+
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class JsonPropertyWriterTest {
+
+  @Test
+  public void shouldCreateMissingObjects() {
+    final String[] paths = {"1", "2", "3", "4", "5"};
+    final JsonObject object2 = new JsonObject()
+      .put("21", 2.1)
+      .put("2", new JsonObject());
+
+    final JsonObject object = new JsonObject()
+      .put("11", "1.1")
+      .put("1", object2);
+
+    writeByPath(object, "5", paths);
+
+    assertThat(object, allOf(
+      hasJsonPath("1.2.3.4.5", "5"),
+      hasJsonPath("11", "1.1"),
+      hasJsonPath("1.21", 2.1)
+    ));
+  }
+
+  @Test
+  public void shouldWriteStringByPath() {
+    final String[] paths = {"1", "2", "3", "4", "5"};
+    final String expectedString = "A string";
+
+    final JsonObject object = new JsonObject();
+    writeByPath(object, expectedString, paths);
+
+    assertThat(object, hasJsonPath("1.2.3.4.5", "A string"));
+  }
```

Comment:
As this example is a simpler example than the one above, I might be tempted to put it first in the class

Reply:
I just realized that it is a duplicate of the 1st test. I'll remove it.